File: CFS/1950A.py
def solve():
    a,b,c=map(int,input().split())

    if a<b and b<c:
        print("STAIR")
    elif a<b and b>c:
        print("PEAK")
    else:
        print("NONE")


t=int(input())
for _ in range(t):
    solve()

File: CFS/1950A.cpp
#include <bits/stdc++.h>
using namespace std;

void solve(int a, int b, int c)
{
  if (a < b && b < c)
    cout << "STAIR" << endl;
  else if (a < b && b > c)
    cout << "PEAK" << endl;
  else
    cout << "NONE" << endl;
}

int main()
{
  ios_base::sync_with_stdio(false);
  cin.tie(0); cout.tie(0);
  int t; cin >> t;
  while (t--)
  {
    int a, b, c; cin >> a >> b >> c;
    solve(a, b, c);
  }
  return 0;
}